File: day1/Point.java
import java.lang.Math;
class Point
{
    int xCoordinate,yCoordinate;
    Point(int x,int y)
    {
        this.xCoordinate=x;
        this.yCoordinate=y;
    }
    public static double getEuclideanDistance(Point p1,Point p2)
    {
        int sum=(p2.xCoordinate-p1.xCoordinate)*(p2.xCoordinate-p1.xCoordinate)+(p2.yCoordinate-p1.yCoordinate)*(p2.yCoordinate-p1.yCoordinate);
        return(Math.sqrt(sum));
    }
}

File: day1/point.py
class Point:
    def __init__(self,x,y):
        self.x=x
        self.y=y

    def getEuclideanDistance(p1,p2):
        return(((p2.x-p1.x)**2+(p2.y-p1.y)**2)**0.5)

p1=Point(0,0)
p2=Point(0,2)
print(Point.getEuclideanDistance(p1,p2))